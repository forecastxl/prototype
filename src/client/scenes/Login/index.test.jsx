import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import { DumbLogin } from './index'

describe('<DumbLogin />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<DumbLogin session={{}} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })

  it('renders client errors', () => {
    const session = { errorName: 'ClientError', errorMessage: 'Client error' }
    const wrapper = shallow(<DumbLogin session={session} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })

  it('renders server errors', () => {
    const session = { errorName: 'ServerError', errorMessage: 'Server error' }
    const wrapper = shallow(<DumbLogin session={session} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
