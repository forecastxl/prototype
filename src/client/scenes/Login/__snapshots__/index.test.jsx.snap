// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders client errors 1`] = `
<ClientError
  errorMessage="Client error"
/>
`;

exports[`<Login /> renders correctly 1`] = `<ReduxForm />`;

exports[`<Login /> renders server errors 1`] = `
<ServerError
  errorMessage="Server error"
/>
`;
